prisma/schema.prisma

```diff
@@ -15,12 +15,29 @@ datasource db {
 }
 
 model User {
-  id      String @id @default(uuid())
-  clerkId String @unique
+  id       String @default(uuid())
+  clerkId  String
+  tenant   Tenant @relation(fields: [tenantId], references: [id])
+  tenantId String
 
   email String?
   name  String
 
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now()) @updatedAt
+
+  @@id([id, tenantId])
+  @@unique([clerkId, tenantId])
+}
+
+model Tenant {
+  id        String   @id @default(uuid())
+  name      String
+  slug      String   @unique
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @updatedAt @map("updated_at")
+  clerkId   String   @unique
+  User      User[]
+
+  @@map("tenant")
```

Comment:
it should default to this table name, no caps. If we want to specify it anyway as a redundancy then lets do it for all tables?